general/mem_manager.hpp

```diff
@@ -112,11 +112,22 @@ MemoryClass operator*(MemoryClass mc1, MemoryClass mc2);
     In many respects this class behaves like a pointer:
     * When destroyed, a Memory object does NOT automatically delete any
       allocated memory.
-    * Only the method Delete() will deallocate a Memory object.
-    * Other methods that modify the object (e.g. New(), Wrap(), etc) will simply
-      overwrite the old contents.
-    * One difference with a pointer is that a const Memory object does not allow
-      modification of the content (unlike e.g. a const pointer).
+    * Only the method `Delete()` will deallocate a Memory object.
```

Comment:
Typically, when we refer to methods/classes (which auto-generates links), we do not put them in back-quotes, see e.g. here: https://github.com/mfem/mfem/blob/3c1859e1d7d9d7bb35ed3a5261f15dcc24343d9a/linalg/operator.hpp#L65-L66

This is consistent with how Doxygen displays the methods/classes themselves: see e.g. the "Public Member Functions" section at https://docs.mfem.org/4.4/classmfem_1_1Operator.html

Reply:
That said, this section is actually not formatted as a bullet list by doxygen -- to get that we need to re-format this a bit, namely replace `*` with `-` since the first `*` in this type of comments is ignored.

Reply:
done in [418054b](https://github.com/mfem/mfem/pull/2355/commits/418054b770a4cac104a33b46f18dce432a3f5482)